### PR TITLE
epic/COR-1475-safety-region-removal

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard25**
## Removal of Safety Regions
This PR will remove the Safety Regions from the dashboard. The work in this PR is done in many smaller steps in previous PRs:

* COR-1505-remove-vr-from-sitemap by @VWSCoronaDashboard18 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4699
* COR-1492-site-navigation-VR-removal by @VWSCoronaDashboard18 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4700
* COR-1485-remove-VR-from-GM-sidebar by @VWSCoronaDashboard18 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4698
* COR-1495-remove-the-VR-sidebar by @VWSCoronaDashboard18 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4702
* COR-1480-remove-from-footer by @VWSCoronaDashboard18 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4703
* COR-1488-remove-searchbar by @VWSCoronaDashboard18 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4701
* feat: COR-1509-removed Gedrag archived section by @VWSCoronaDashboard27 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4704
* COR-1503: Removed vr-related url by @VWSCoronaDashboard27 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4712
* feature/COR-1500-e2e-package-removal by @VWSCoronaDashboard26 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4710
* feature/COR-1481-remove-safety-regions-page by @VWSCoronaDashboard21 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4709
* feat: COR-1486-toggle-vr-references-removal by @VWSCoronaDashboard27 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4714
* feature/COR-1476-non-clickable-choropleth by @VWSCoronaDashboard21 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4715
* COR-1515-remove-VR-pages-part-II by @VWSCoronaDashboard18 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4705
* COR-1489-remove-toggle-view-on-patients-page by @VWSCoronaDashboard18 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4717
* feature/COR-1502-remove-redirects-vr-pages by @VWSCoronaDashboard21 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4706
* feature/COR-1482-safety-region-page-removal by @VWSCoronaDashboard26 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4707
* COR-1487-remove-toggle-on-positive-tested-page by @VWSCoronaDashboard18 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4716
* feature/COR-1510-non-clickable-choropleth by @VWSCoronaDashboard21 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4720
* feature/COR-1479-contact-tracing-page-removal by @VWSCoronaDashboard26 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4713
* feature/COR-1478-non-clickable-choropleth by @VWSCoronaDashboard26 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4721
* COR-1499-safety-region-removal-navigation-and-router by @VWSCoronaDashboard18 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4724
* Feature/cor 1498 vaccinations toggle button removal cherrypick by @VWSCoronaDashboard27 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4728
* feature/COR-1546-municipality-sidebar-spacing-optimisations by @VWSCoronaDashboard26 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4732
* bugfix/COR-1555-fix-whitespace-add-titles by @VWSCoronaDashboard21 in https://github.com/minvws/nl-covid19-data-dashboard/pull/4735

### CODE REVIEW

**Review by VWSCoronaDashboard26 — DISMISSED** [review]

File: packages/app/src/domain/vaccine/vaccinations-kpi-header.tsx [hunk 1]
```diff
@@ -29,7 +28,7 @@ interface VaccineKpiHeaderProps {
 
 export function VaccinationsKpiHeader({ text, dateUnix, dateOfInsertionUnix }: VaccineKpiHeaderProps) {
   return (
-    <Box paddingTop="40px" borderTopWidth={space[2]} borderColor="gray3" borderStyle="solid">
+    <Box paddingTop="40px" borderTopWidth="2px" borderColor="gray3" borderStyle="solid">
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/domain/vaccine/vaccinations-kpi-header.tsx:31]: This `borderColor="gray3"` should technically be `borderColor={colors.gray3}`.

File: packages/app/src/domain/vaccine/vaccine-coverage-choropleth.tsx [hunk 2]
```diff
@@ -70,64 +64,40 @@ export const VaccineCoverageChoropleth = ({ data }: VaccineCoverageChoroplethPro
...
-          }}
-          dataOptions={{
-            isPercentage: true,
-            getLink: (vrcode) => reverseRouter.vr.vaccinaties(vrcode),
-          }}
-          formatTooltip={(context) => <ChoroplethTooltip data={context} ageGroups={matchingAgeGroups[selectedCoverageKind]} selectedCoverageKind={selectedCoverageKind} />}
-        />
-      )}
+      <DynamicChoropleth
+        map={'gm'}
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/domain/vaccine/vaccine-coverage-choropleth.tsx:91]: The quotes around `gm` are not necessary.

File: packages/app/src/pages/landelijk/positief-geteste-mensen.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { colors, NlTestedOverallValue, TimeframeOption, TimeframeOptionsList } f
 import { GgdTesten } from '@corona-dashboard/icons';
 import { GetStaticPropsContext } from 'next';
 import { useState } from 'react';
-import { RegionControlOption } from '~/components/chart-region-controls';
+import { Box } from '~/components/base';
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/pages/landelijk/positief-geteste-mensen.tsx:5]: `components/base` is a barrel file. It should technically be `components/base/box` given that we opted to not use these kinds of files anymore.

File: packages/app/src/pages/landelijk/thuiswonende-ouderen.tsx [hunk 4]
```diff
@@ -211,9 +207,7 @@ function ElderlyAtHomeNationalPage(props: StaticProps<typeof getStaticProps>) {
                   positive_tested_daily_per_100k: formatNumber,
                 },
               }}
-              dataOptions={{
-                getLink: reverseRouter.vr.thuiswonendeOuderen,
-              }}
+              dataOptions={{}}
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/pages/landelijk/thuiswonende-ouderen.tsx:210]: Can this not be `dataOptions={}`?

File: packages/app/src/pages/landelijk/vaccinaties.tsx [hunk 5]
```diff
@@ -152,6 +153,10 @@ function VaccinationPage(props: StaticProps<typeof getStaticProps>) {
 
   const lastInsertionDateOfPage = getLastInsertionDateOfPage(data, pageMetrics);
 
+  const variables = {
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/pages/landelijk/vaccinaties.tsx:156]: I think this variable name is way too generic, it tells me nothing about how its going to be used. I would highly suggest amending it to something more descriptive.

File: packages/app/src/utils/use-scoped-warning.ts [hunk 6]
```diff
@@ -8,11 +8,7 @@ import { replaceVariablesInText } from '~/utils/replace-variables-in-text';
 export function useScopedWarning(current: string, warning: string) {
   const { commonTexts } = useIntl();
   const scopedGmName = commonTexts.gemeente_index.municipality_warning;
-  const scopedGm = gmData.find(
-    (gm) =>
-      gm.name === scopedGmName ||
-      (gm.searchTerms && gm.searchTerms.includes(scopedGmName))
-  );
+  const scopedGm = gmData.find((gm) => gm.name === scopedGmName || (gm.searchTerms && gm.searchTerms.includes(scopedGmName)));
   const scopedVr = vrData.find((vr) => vr.code === scopedGm?.vrCode);
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/utils/use-scoped-warning.ts:12]: Is this reference to VR still needed?

File: packages/common/src/types/data-utils.ts [hunk 7]
```diff
@@ -7,17 +7,21 @@ import { Gm, Nl, Vr } from '.';
 export type ScopedData = {
   gm: Gm;
   nl: Nl;
+  /** TODO: when this gets removed, also remove the type "DataScopeKeyForRouter".
```
Comment (VWSCoronaDashboard28) [inline packages/common/src/types/data-utils.ts:10]: Has this information been added to the relevant ticket? If not, can it be added?

**Review by VWSCoronaDashboard28 — APPROVED** [review]
I have left some comments, all of which are non blocking. One of them refers to a reference to VR's which may not be needed anymore. But this is also not necessarily a blocker. Therefore, I'm preemptively approving this.

**Review by VWSCoronaDashboard18 — DISMISSED** [review]
the file `packages/app/src/components/breadcrumbs/logic/use-breadcrumbs.tsx` still has "veiligheidsregio" mentioned on lines 107, 117, & 120.

File: packages/app/src/components/choropleth/logic/thresholds.ts [hunk 8]
```diff
@@ -385,11 +322,8 @@ export const thresholds: Thresholds = {
     vaccinated_percentage_60_plus: vaccineCoveragePercentageThresholds,
   },
   vr: {
```
Comment (VWSCoronaDashboard18) [inline packages/app/src/components/choropleth/logic/thresholds.ts:324]: I thought we would only need vulnerable people. A lot of them could go like `autumn_2022_vaccinated_percentage` or `travel`

**Review by VWSCoronaDashboard18 — COMMENTED** [review]
The file `packages/app/src/components/layout/app-content.tsx` on line 27 still mentions 'safety regions'

File: packages/app/src/pages/gemeente/[code]/positief-geteste-mensen.tsx
Comment (VWSCoronaDashboard18) [inline packages/app/src/pages/gemeente/[code]/positief-geteste-mensen.tsx:1]: On `line 58` a mention of VR:

File: packages/app/src/components/choropleth/logic/thresholds.ts (hunk 8 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/components/choropleth/logic/thresholds.ts:324]: As far as I know only these should be here:
- infected_locations_percentage
- positive_tested_daily_per_100k

Comment (VWSCoronaDashboard18) [reply packages/app/src/components/choropleth/logic/thresholds.ts:324]: Non blocking, so I will resolve for now.

**Review by VWSCoronaDashboard26 — APPROVED** [review]

**Review by VWSCoronaDashboard28 — APPROVED** [review]

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Because of the follow-up/cleanup ticket solving the comments, Here is an approval for this PR!

**Pull request merged**